.github/workflows/update-serial.yml

```diff
@@ -0,0 +1,44 @@
+name: Update and Tag DNS Zone Serial
+
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    name: "Release"
```

Comment:
nit: Hier könnte man den  Namen noch anpassen (das steht in der Action weiter unten, ist von daher nicht ganz so kritisch)